manifests/plugin/mysql.pp

```diff
@@ -4,6 +4,7 @@
   $ensure           = present,
   $manage_package   = $collectd::manage_package,
   $interval         = undef,
+  $databases        = { },
```

Comment:
create_resources takes all of that away from us
it does nothing on empty hash, and fails horribly if passed anything other than a (valid) hash.